react/src/pages/analysis/Analysis.tsx

```diff
@@ -152,48 +198,92 @@ export default function Analysis() {
             <Container maxWidth="lg" className={classes.container}>
                 <MaterialTable
                     columns={[
-                        { title: 'AnalysisID', field: 'analysisID', type: 'string' },
-                        { title: 'Submitted', field: 'dateSubmitted', type: 'string' },
-                        { title: 'Submitted By', field: 'submittedBy', type: 'string' },
-                        { title: 'Project', field: 'project', type: 'string' },
-                        { title: 'Participants', field: 'participants', type: 'string', render: rowData => rowData.participants.join(', ') },
-                        { title: 'Pipeline', field: 'pipeline', type: 'string' },
-                        { title: 'Status', field: 'status', type: 'string' }
+                        { title: 'Analysis ID', field: 'analysis_id', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Pipeline ID', field: 'pipeline_id', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Assignee ID', field: 'assignee', type: 'string', editable: 'never', width: '8%' },
+                        { title: 'Requester ID', field: 'requester', type: 'string', editable: 'never', width: '8%' },
```

Comment:
Get rid of `ID` in the display name for the latter three to save some display space.